lib/src/code_field/search_result_highlighted_builder.dart

```diff
@@ -4,7 +4,7 @@ import '../search/result.dart';
 import '../search/search_navigation_state.dart';
 
 @visibleForTesting
-const matchBackgroundColor = Color.fromARGB(141, 255, 235, 59);
+const matchBackgroundColor = Colors.green;
```

Comment:
Let's use Chrome's colors instead as it is the most popular browser, and the people are used to it.
![image](https://github.com/akvelon/flutter-code-editor/assets/44893228/da87e09c-ba7f-40d4-80d8-de1425eda5b3)